vmdb/app/views/shared/dialogs/_dialog_field_drop_down_list.haml

```diff
@@ -26,8 +22,14 @@
         $.each(data.values.refreshed_values, function(index, value) {
           var option = '<option ';
           option += 'value="' + value[0] + '" ';
-          if (data.values.checked_value.toString() === value[0].toString()) {
-            option += 'selected="selected" ';
+          if (data.values.checked_value !== null) {
+            if (data.values.checked_value.toString() === value[0].toString()) {
+              option += 'selected="selected" ';
+            }
+          } else {
+            if (index === 0) {
+              option += 'selected="selected" ';
+            }
```

Comment:
Right now I think the effort to convert this whole thing to Angular would be pretty high, there's still a lot of stuff being done in the controller and the session and edit variables. Otherwise, I'd love for this kind of mess to be cleaned up.